tests/test_exceptions.py

```diff
@@ -0,0 +1,42 @@
+from crawler.exceptions import (
```

Comment:
Thanks for adding these tests!

Reply:
The code coverage went down. I wouldn't have been able to merge without them 😆